WhaleSpotting/react-app/src/components/Button.tsx

```diff
@@ -0,0 +1,55 @@
+import React from "react";
+import "../styles/Buttons.scss";
+import { Link } from "react-router-dom";
+
+enum Style {
```

Comment:
We should export this and then can use `Style.primary` instead of 0 etc when calling it :)